hoomd/hpmc/pytest/test_compute_sdf.py

```diff
@@ -0,0 +1,35 @@
+import hoomd
+import pytest
+import numpy as np
+
+
+def test_before_attaching():
+    sdf = hoomd.hpmc.compute.SDF(xmax=0.02, dx=1e-4)
+
+    assert sdf.xmax == 0.02
+    assert sdf.dx == 1e-4
+    assert sdf.sdf is None
+    assert sdf.betaP is None
+
+
+def test_after_attaching(simulation_factory, lattice_snapshot_factory):
```

Comment:
I'd like to see this test repeated for every integrator shape, because the python SDF class needs to use a different cpp class depending on which integrator is being used for the simulation.